client/app/email-domain/email/email-domain-email.controller.js

```diff
@@ -123,6 +125,11 @@ angular
             this.$window.open(this.webMailUrl, "_blank");
         }
 
+
+        openOMMTab () {
```

Comment:
Can we use a link instead of button + ngClick + $window.open ?

It's just a `<a href="..." target="_blank"></a>` ... Don't forget title attribute title attribute for accessibility if needed :)

Reply:
I just followed example from line [ #124 ](https://github.com/miecio45/ovh-manager-web/blob/0d8a8d9afa9ee1f0ce392b81e6931521b329d34a/client/app/email-domain/email/email-domain-email.controller.js#L124) for webmail page link and assume that this is the model solution

Reply:
Yes we know our code and its weak points.

Currently, we are improving our source code to improve the product. I invite you to follow my recommendations.

Reply:
I hope that I fix it :)